modules/fts/pages/fts-searching-with-the-rest-api.adoc

```diff
@@ -100,6 +100,41 @@ http://localhost:18094/api/index/travel-sample-index/query \
 For additional information on secure communication with Couchbase Server, see xref:manage:manage-security/manage-connections-and-disks.adoc[Manage Connections and Disks].
 
 
+[#searching-with-request-timeouts]
+== Searching with timeouts
+
+Once the the index becomes really large, it is possible that certain search requests may take
+longer time to finish the execution. Such slow responses would affect the query latencies as well as the resource consumption on the server side.
+
+In order to handle such situations predictably, users can pass a search "timeout" value in the request.
+By specificing a timeout parameters in the seach request, users are bound to get a response within the stipulated time.
+Here the timeout value is specified in milliseconds.
+
+If the search request times out before gathering all the results, then the partial results will be
+returned to the user with detailed partition level search result status.
+In case of timeouts, the response's http status code would be 412(Precondition Failed).
+
+For example:
+
+[source,bourne]
+----
+curl -u Administrator:password -XPOST -H "Content-Type: application/json" \
+http://localhost:18094/api/index/travel-sample-index/query \
+-d '{
+  "ctl": {
+    "timeout": 7000
+  },
+  "query": {
+    "query": "{ \"+nice +view\" }"
+  }
+}'
+----
+
+Please note that the default value of timeout is `10` seconds and users may override this
```

Comment:
Note that... and you may change this value to suit your use case.

(General recommendations:
- avoid using "please" 
- write in second person i.e., refer to users as "you" )